templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -2,49 +2,73 @@ param name string
 param location string = resourceGroup().location
 param tags object = {}
 
+@description('Allowed origins')
+param allowedOrigins array = []
+
+@description('Name of the environment for container apps')
 param containerAppsEnvironmentName string
-param containerName string = 'main'
-param containerRegistryName string = ''
 
-@allowed([ 'Single', 'Multiple' ])
-param revisionMode string = 'Single'
+@description('CPU cores allocated to a single container instance, e.g., 0.5')
+param containerCpuCoreCount string = '0.5'
 
-@description('Minimum number of replicas to run')
-@minValue(1)
-param containerMinReplicas int = 1
-@description('Maximum number of replicas to run')
+@description('The maximum number of replicas to run. Must be at least 1.')
 @minValue(1)
 param containerMaxReplicas int = 10
 
-param secrets array = []
+@description('Memory allocated to a single container instance, e.g., 1Gi')
+param containerMemory string = '1.0Gi'
+
+@description('The minimum number of replicas to run. Must be at least 1.')
+param containerMinReplicas int = 1
+
+@description('The name of the container')
+param containerName string = 'main'
+
+@description('The name of the container registry')
+param containerRegistryName string = ''
+
+@description('The protocol used by Dapr to connect to the app, e.g., http or grpc')
+@allowed([ 'http', 'grpc' ])
+param daprAppProtocol string = 'http'
+
+@description('The Dapr app ID')
+param daprAppId string = containerName
+
+@description('Enable Dapr')
+param daprEnabled bool = false
+
+@description('The environment variables for the container')
 param env array = []
+
+@description('Specifies if the resource is external')
```

Comment:
This is part of the ingress section and actually means whether the ingress is exposed externally or not.